docs-ui/components/layout-thirds.stories.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import Breadcrumbs from 'app/components/breadcrumbs';
+import * as Layout from 'app/components/layouts/thirds';
+
+const crumbs = [
+  {
+    label: 'Issues',
+    to: '',
+  },
+  {
+    label: 'List',
+    to: '',
+  },
+  {
+    label: 'Detail',
+    to: '',
+  },
+];
+
+storiesOf('Layouts|Thirds', module)
+  .add(
+    '66/33 layout',
+    withInfo('Two column layout with header & sidebar')(() => (
+      <Container>
+        <Layout.Header>
+          <Heading>Some heading content</Heading>
+        </Layout.Header>
+        <Layout.LayoutBody>
```

Comment:
Please why did you decide to name it "Layout.LayoutBody" instead of "Layout.Body"?

Reply:
I can rename.